src/platform-integration/android/splash-screen.md

```diff
@@ -82,6 +88,8 @@ background color of the Flutter UI.
 </style>
 ```
 
+[drawables]: {{site.android-dev}}/studio/write/resource-manager#import
```

Comment:
That link seems to talk about how to import Drawables into an Android project using Android Studio.

Maybe https://developer.android.com/guide/topics/resources/drawable-resource is a better URL to send developers to?

Reply:
I saw that page, too. I thought maybe adding it to the project was needed? I mean, a Drawable itself is pretty normal.

Reply:
Ahh, Okiday